src/pages/MonthlyGood/MonthlyGoodSetupPage.vue

```diff
@@ -8,174 +8,219 @@
+										/>
+									</div>
+
+									<div class="small-12 columns">
+										<ul class="text-right validation-errors" v-if="$v.donation.$invalid">
+											<li v-if="!$v.donation.minValue || !$v.donation.maxValue">
+												Enter an amount of $0-$10,000
+											</li>
+										</ul>
+									</div>
+								</div>
+
+								<div class="row text-left">
+									<div class="columns">
+										<strong class="additional-left-pad-spans">Total/month</strong>
+									</div>
+
+									<div class="medium-5 small-6 columns">
+										<strong
+											class="additional-left-pad-currency"
+										>{{ totalCombinedDeposit | numeral('$0,0.00') }}</strong>
+									</div>
+
+									<div class="small-12 columns">
+										<ul class="text-center validation-errors"
+											v-if="!$v.mgAmount.maxTotal || !$v.donation.maxTotal"
+										>
+											<li>
+												The maximum Monthly Good total is $10,000.<br>
+												Please try again by entering in a smaller amount.
+											</li>
+										</ul>
 									</div>
 								</div>
-							</div>
 
+								<div class="row text-left">
+									<div class="small-12 columns">
+										<div class="additional-left-pad-spans">
+											Select a category to focus your lending
+											<kv-dropdown-rounded v-model="selectedGroup" class="group-dropdown">
+												<option
+													v-for="(option, index) in lendingCategories"
+													:value="option.value"
+													:key="index"
+												>
+													{{ option.label }}
+												</option>
+											</kv-dropdown-rounded>
+										</div>
+									</div>
+								</div>
 
-							<div class="row small-collapse">
-								<div class="small-12 columns">
-									<em class="text-center">Rest easy, you can cancel anytime.</em>
+								<div class="row small-collapse">
+									<div class="small-12 columns">
+										<em class="text-center">Rest easy, you can cancel anytime.</em>
+									</div>
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="row align-center text-center">
+						<div class="large-9 medium-10 small-12 columns">
+							<p>
+								<!-- eslint-disable-next-line max-len -->
+								<strong><em>We'll charge your PayPal account each month, and any credit in your Kiva account will be automatically re-lent for you.</em></strong>
+							</p>
+							<p v-if="hasAutoDeposits">
+								<em>* Your new Monthly Good contribution will replace your existing auto deposit.</em>
+							</p>
+							<p v-if="hasAutoLending">
+								<!-- eslint-disable-next-line max-len -->
+								<em>* Enrolling in Monthly Good will also disable your current auto lending settings.</em>
+							</p>
+							<div v-if="hasBillingAgreement">
+								<kv-button
+									type="submit"
+									data-test="confirm-monthly-good-button"
+									class="smaller"
+									:disabled="$v.$invalid"
+									@click.native="submitMonthlyGood()"
```

Comment:
You can put this submit event on the form instead so that it gets called no matter how the form is submitted (i.e. clicking the submit button, hitting enter when the form has focus, any control methods used by screen readers)

Reply:
My thinking was that if the user doesnt have a billing agreement, I dont want to fire the event on form submit, but only after the paypal component has done its thing and emitted the event.

Reply:
I get where @emuvente is coming from and you could do that conditionally for the case of the "Confirm" button. I'm guessing someone tabbing to the PayPal button and clicking "enter" would not trigger the form submit as it's in an iframe...

Reply:
@eddieferrer You could probably handle that by checking `hasBillingAgreement` before running the mutation